modules/preparing-aws-sts-credentials-for-oadp.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-aws-sts/oadp-aws-sts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="preparing-aws-sts-credentials-for-oadp_{context}"]
+= Preparing AWS STS credentials for OADP
+
+An {aws-full} account must be prepared and configured to accept an {oadp-first} installation. Prepare the {aws-short} credentials by following the proceeding steps.
+
+.Procedure
+. Define the `cluster_name` environment variable by running the following command:
++
+[source,terminal]
+----
+$ export CLUSTER_NAME= <AWS cluster name> <1>
+----
+<1> The variable can be set to any value.
+
+. Retrieve all the details of the `cluster` such as the `AWS_ACCOUNT_ID, OIDC_ENDPOINT` by running the following command:
++
+[source,terminal]
```

Comment:
There is a space between all of these except the last two. Should you add a space there>?

Reply:
I'm think it's OK like this.